infrastructure/evault-core/src/w3id/w3id.ts

```
@@ -7,64 +7,64 @@ import { SecretsStore } from "../secrets/secrets-store";
-      W3ID.secretsStore = new SecretsStore(
-        process.env.SECRETS_STORE_PATH!,
-        options.password
-      );
-    }
+        // Initialize secrets store if not already done
+        if (!W3ID.secretsStore) {
+            if (!options.password) {
+                throw new Error("Password is required for secrets store");
+            }
+            W3ID.secretsStore = new SecretsStore(
+                process.env.SECRETS_STORE_PATH!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)